transplant/transplant_master.py

```diff
@@ -523,6 +525,11 @@ def __init__(self, executable='matlab', arguments=tuple(), msgformat='msgpack',
                 arguments += '-minimize',
         if not jvm and '-nojvm' not in arguments:
             arguments += '-nojvm',
+        paths = (os.path.dirname(__file__))
+        if addpath is not None:
```

Comment:
The default value is an empty list, so comparing to `None` seems incorrect here. In fact you can probably drop the entire line, as `for path in addpath` will not execute anything for an empty list anyway.

Reply:
you are right